pkg/manager/manager.go

```diff
@@ -374,6 +374,11 @@ func (m MediaManager) ListQualityDefinitions(ctx context.Context) ([]*model.Qual
 	return m.storage.ListQualityDefinitions(ctx)
 }
 
+// ListQualityDefinitions list stored quality definitions
```

Comment:
I think this comment is wrong right?

Reply:
fixed the comments, I end up copy pasting a lot of these and forget about the comment